src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz.erl

```diff
@@ -123,6 +123,7 @@ create_entity_if_authorized(Context, OrgId, CreatorAId, ObjectType) ->
             {error, forbidden};
```

Comment:
This will still fail with a 500 in case of a non-forbidden error return, which I think is correct - but it does mean that we'll potentially lose logging/error info here.  Let's create a new issue to confirm & fix that if needed.

Reply:
Yeah - I think it would be useful to add the matcher for the {error, Error} case from https://github.com/chef/chef-server/blob/170469aeff9eb81eb98784a2d033f9361ee5012a/src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz.erl#L332

Reply:
and similarly a matcher for {error, Error} in https://github.com/chef/chef-server/blob/170469aeff9eb81eb98784a2d033f9361ee5012a/src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz.erl#L118

Reply:
That error would also need to propogate all the way up through https://github.com/chef/chef-server/blob/6d31841ecd73d984d819244add7ad6ebac284323/src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_organizations.erl#L253, https://github.com/chef/chef-server/blob/6d31841ecd73d984d819244add7ad6ebac284323/src/oc_erchef/apps/oc_chef_wm/src/oc_chef_associations.erl#L212, https://github.com/chef/chef-server/blob/afd0ec44666facb2c9c87d937fcbce7bb57ad10f/src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl#L367 into the modules calling those functions.